tests/darwin/exporter/formats/export_nifti_test.py

```diff
@@ -4,19 +4,50 @@
 
 import nibabel as nib
 import numpy as np
+
 from darwin.exporter.exporter import darwin_to_dt_gen
 from darwin.exporter.formats import nifti
 from tests.fixtures import *
 
 
-def test_video_annotation_nifti_export_v2(team_slug: str):
+def test_video_annotation_nifti_export_single_slot(team_slug: str):
```

Comment:
Is there a potential sad path test and where we can test it errors as expected?

Reply:
The data used to test it works as expected/errors is in the data.zip folder.